fix_proto_conf_desc.py

```diff
@@ -0,0 +1,170 @@
+import ida_segment
+import idautils
+import idc
+import ida_bytes
+import ida_name
+from ida_nalt import REF_OFF32, REFINFO_SIGNEDOP, REFINFO_SELFREF
+from idc import FF_DATA, FF_DWORD
+
+class CStruct: 
+    SELFREF = REFINFO_SIGNEDOP | REFINFO_SELFREF
+
+    def __init__(self):
+        self.tif = 0
```

Comment:
`self.tif` is not of type `int` so this assignment makes no sense. 
```suggestion
        self.tif = ida_typeinf.tinfo_t()
```